tendrl-gluster-integration.spec

```diff
@@ -21,6 +21,7 @@ Requires: python-gevent
 Requires: python-greenlet
 Requires: pytz
 Requires: tendrl-common
+Requires: tendrl-node-agent
```

Comment:
tendrl-node-agent is always supposed to be present before tendrl-<sds>-integration, please remove this